manifests/plugin/post_transaction_actions.pp

```diff
@@ -39,5 +45,7 @@
       content => "# Puppet maintained actions\n# \$key:\$state:\$command\n\n",
       order   => '01',
     }
+
+    create_resources('yum::post_transaction_action', $actions)
```

Comment:
please don't use the legacy create_resources. Instead iterate on the hash and use the splash operator for the resources. An example is here: https://github.com/voxpupuli/puppet-ferm/blob/master/manifests/init.pp#L88-L92

Reply:
Done